pkg/activator/net/throttler_test.go

```diff
@@ -398,10 +399,18 @@ func TestThrottlerSuccesses(t *testing.T) {
 			revisions.Informer().GetIndexer().Add(tc.revision)
 
 			updateCh := make(chan revisionDestsUpdate)
-			defer close(updateCh)
 
 			throttler := NewThrottler(ctx, "130.0.0.2")
-			go throttler.run(updateCh)
+			var grp errgroup.Group
+			grp.Go(func() error { throttler.run(updateCh); return nil })
```

Comment:
if you aren't using the error functionality, then why not `sync.WaitGroup`?  (same below)

Reply:
more consice, imo, than
```
var wg sync.WaitGroup{}
wg.Add(1)
go func() {
  defer wg.Release()
  throttler.run(chan)
}()
```

Reply:
3 vs 6 lines.

Reply:
For a moment I had thought WaitGroup had a closer signature, but had clearly repressed the Add/Release nonsense.  I agree this is much better 🙄

Reply:
😄